### PR TITLE
Update elementwise.py

### PR DESCRIPTION
**Pull request by Paulyn01**

### CODE REVIEW

**Comment by simonetgordon** [comment]
Hi Pauline, 

Just some tiny changes to be made, then it's good to be merged!

File: ivy/functional/ivy/elementwise.py [hunk 1]
```diff
@@ -4038,6 +4038,89 @@ def positive(
...
+    >>> print(y)
+    ivy.array([2, 3, 5, 7])
+
+    >>> x = ivy.array([0,-1,-0.5,2,3])
+    >>> y = ivy.zeros(5)
+    >>> ivy.positive(x,out=y)
+    >>> print(y)
+    ivy.array([0., -1., -0.5,  2.,  3.])
+
+    >>> x = ivy.array([[1.1,2.2,3.3], \
```
Comment (simonetgordon) [inline ivy/functional/ivy/elementwise.py:4057]: To clarify, i mean keep the spacing consistent in all the array lists, e.g. `>>> x = ivy.array([[1.1,2.2,3.3], \` should be `>>> x = ivy.array([[1.1, 2.2, 3.3], \`

File: ivy/functional/ivy/elementwise.py [hunk 2]
```diff
@@ -3796,7 +3796,7 @@ def not_equal(
     Functional Examples
     ------------------
 
-    With :code:`ivy.Array` inputs:
+    With: code:`ivy.Array` inputs:
```
Comment (simonetgordon) [inline ivy/functional/ivy/elementwise.py:3799]: The colons should surround 'code' as before, like `With :code:`ivy.Array` inputs:`

**Review by simonetgordon — CHANGES_REQUESTED** [review]

File: ivy/functional/ivy/elementwise.py (hunk 2 above)
Comment (Paulyn01) [reply ivy/functional/ivy/elementwise.py:3799]: hi could you pleas assist me why then docstring test is failing

**Review by simonetgordon — COMMENTED** [review]
Making changes myself

**Comment by simonetgordon** [comment]
Co-authored-by: @simonetgordon <simonegordon12@icloud.com>

**Pull request merged**